server/routes/shared_api/observation/series.py

```diff
@@ -64,6 +79,21 @@ def series_within():
   variables = list(filter(lambda x: x != "", request.args.getlist('variables')))
   if not variables:
     return 'error: must provide a `variables` field', 400
+  batch_size = request.args.get('batch_size') or _MAX_BATCH_SIZE
```

Comment:
nit: usually we want request arguments to be camelCase

Reply:
Switched to camelCase